packages/okta-react/README.md

```diff
@@ -301,7 +399,7 @@ export default App;
 
 #### PKCE Example
 
-Assuming you have configured your application to allow the `Authorization code` grant type, simply pass `pkce=true` to the `Security` component. This will configure the [Auth service][] to perform PKCE flow for both login and token refresh.
+Assuming you have configured your application to allow the `Authorization code` grant type, simply pass `pkce=true` to the `Security` component. This will configure the [Auth service][] to perform PKCE flow for both login and token refresh.  
```

Comment:
This is fine, I don't actually see any changes here, so no need to change it in this PR. but I think PKCE is now our default and implicit is effective deprecated. This is what the admin UI looks like now. 
<img width="699" alt="Screen Shot 2020-02-28 at 12 39 10 PM" src="https://user-images.githubusercontent.com/47947204/75585702-8c29de80-5a27-11ea-9276-da769d2083cb.png">